.circleci/config.yml

```diff
@@ -1256,17 +1318,17 @@ jobs:
             equal: [<< parameters.binary_type >>, "solcjs"]
           steps:
             - run:
-                name: External << parameters.project >> tests (solcjs)
+                name: External << parameters.project >> tests (<< parameters.binary_type >>)
                 command: |
-                  test/externalTests/<< parameters.project >>.sh solcjs /tmp/workspace/soljson.js
+                  test/external_tests.py test --solc-binary-type << parameters.binary_type >> --solc-binary-path /tmp/workspace/soljson.js --run << parameters.project >>
       - when:
           condition:
             equal: [<< parameters.binary_type >>, "native"]
           steps:
             - run:
-                name: External << parameters.project >> tests (native)
+                name: External << parameters.project >> tests (<< parameters.binary_type >>)
                 command: |
-                  test/externalTests/<< parameters.project >>.sh native /tmp/workspace/solc/solc-static-linux
+                  test/external_tests.py test --solc-binary-type << parameters.binary_type >> --solc-binary-path /tmp/workspace/solc/solc-static-linux --run << parameters.project >>
```

Comment:
If you extract this bit into a separate PR, we could merge it right away. It does not depend on Foundry in any way, only on the the external PR refactor, which is now merged, and the only remark I had here were the missing quotes.

Reply:
Done here: https://github.com/ethereum/solidity/pull/14191